Cannot find module 'linebreak'

Hi,

When I run the node application, I get this error:

```
Error: Cannot find module 'linebreak'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/opt/esmasshopping/api/node_modules/pdfmake/src/textTools.js:4:19)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/opt/esmasshopping/api/node_modules/pdfmake/src/docMeasure.js:4:17)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
```

Could be the "linebreak" dependence in package.json?
Thanks.

--- a/package.json
+++ b/package.json
@@ -8,6 +8,7 @@
   },
   "dependencies": {
     "pdfkit": "^0.8.0",
+    "linebreak": "^0.3.0",
     "lodash": "^4.17.3"
   },
   "devDependencies": {
@@ -22,7 +23,6 @@
     "gulp-util": "^3.0.8",
     "gulp-each": "^0.2.0",
     "gulp-file-contents-to-json": "^0.2.1",
-    "linebreak": "^0.3.0",
     "iconv-lite":"^0.4.15",
     "json-loader": "^0.5.4",
     "sinon": "^1.17.7",

